spec.bs

```diff
@@ -2780,8 +2788,19 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
     /fenced-frame/history-length-fenced-navigations-replace-do-not-contribute-to-joint.https.html
     /fenced-frame/history-length-outer-page-navigation-not-reflected-in-fenced.https.html
   </wpt>
+</div>
 
-  Modify step 8 of the same algorithm to include the following condition:
+<div algorithm=navigate>
+  Modify the definition of [[HTML]]'s [=navigate=] algorithm to include an extra parameter: an
+  optional [=string=] |sharedStorageContext| (default null).
```

Comment:
While we're here, notice that `#navigate` has all of its optional non-positional params as _definitions_, so they are linkable. Let's do the same here. Let's make `|sharedStorageContext|` not just a variable but a definition, and then link to it in the one spot we reference it below, later in this algorithm.

Reply:
Turned `|sharedStorageContext|` into a `<dfn>`.